src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
 export class Triangle {
+  shape: Figure['shape'] = 'triangle';
+
+  constructor(
+    public color: Figure['color'],
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.color = color;
```

Comment:
Assigning values to class properties in the constructor is redundant as these properties are already initialized by TypeScript.